packages/playground/src/components/node_selector/TfDomainName.vue

```diff
@@ -56,7 +56,7 @@
                 ]"
                 return-object
               >
-                <template #append-item v-if="pagination.page !== -1">
+                <template #append-item v-if="pagination.page !== -1 && loadedDomains.length === size">
```

Comment:
pagination task should handle the last page by setting the page with -1 so why do we need to check on the size ?